fastai/utils/collect_env.py

```diff
@@ -189,8 +189,9 @@ def check_perf():
     }
     print("\n*** CUDA status")
     if torch.cuda.is_available():
+        pynvml = get_pynvml()
         pynvml.nvmlInit()
-        nvidia_ver = pynvml.nvmlSystemGetDriverVersion().decode('utf-8')
+        nvidia_ver = pynvml.nvmlSystemGetDriverVersion()
```

Comment:
you removed the utf-8 decode call - was it not needed? I don't remember why I put it there in first place

Reply:
I did it, because the pynvx `cudaSystemGetDriverVersion` returns an `int`. In https://github.com/1132719438/pynvx/pull/3 i proposed to change it's return value to string.

Reply:
But nvmlSystemGetDriverVersion returns a string. so yes, pynvx should do the same. So let's not remove that decode call.